### PR TITLE
add KEDA to civo marketplace

### PR DESCRIPTION
**Pull request by dirien**
Thank you for wanting to submit a Pull Request to the Civo Kubernetes Marketplace repository!

If your pull request is to submit a new application to the marketplace, please answer the following questions:

* [x] Have you followed the guidelines in our Contributing document in setting up the application?
* [x] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [x] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
* [x] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?

If your pull request concerns an existing Marketplace application, please make sure you have:

* [ ] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [ ] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [ ] Tested that the application works with the proposed updates applied (including a screenshot)

### install
image

### running pods
image

### uninstall
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Any kubeconfig within the repo - could potentially contain auth data
```
Comment (DoNnMyTh) [inline .gitignore:1]: please remove this file

File: keda/post_install.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+## KEDA
+
+KEDA is a Kubernetes-based Event Driven Autoscaler. With KEDA, you can drive the scaling of any container in Kubernetes based on the number of events needing to be processed.
+
+KEDA is a single-purpose and lightweight component that can be added into any Kubernetes cluster. KEDA works alongside standard Kubernetes components like the Horizontal Pod Autoscaler and can extend functionality without overwriting or duplication. With KEDA you can explicitly map the apps you want to use event-driven scale, with other apps continuing to function. This makes KEDA a flexible and safe option to run alongside any number of any other Kubernetes applications or frameworks.
+
+### Features
+
+- Event-driven 
+- Autoscaling Made Simple
```
Comment (DoNnMyTh) [inline keda/post_install.md:10]: may add some examples aslo

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
...
 kubeconfig*
```
Comment (DoNnMyTh) [inline .gitignore:2]: delete this file

**Review by DoNnMyTh — CHANGES_REQUESTED** [review]
@dirien please do the requested changes

File: keda/post_install.md (hunk 2 above)
Comment (dirien) [reply keda/post_install.md:10]: done, added an example for the Redis autoscaler. As there are many more, the link to the official doc should help too.

**Comment by dirien** [comment]
hi @DoNnMyTh, did the suggested changes. Thanks for your time and feedback 👍🏻

**Comment by DoNnMyTh** [comment]
Hello @dirien Please delete manually https://github.com/civo/kubernetes-marketplace/pull/209/files#diff-e2eb93a61ffd7877ea5c751abcb3a618e8e2e9a2073a27f66d4114fe10819f86

**Comment by dirien** [comment]
hi @DoNnMyTh, removed the .DS_Store file from my commit. So only my changes (and removed gitignore) are in this PR.

**Review by DoNnMyTh — APPROVED** [review]
Can be merged now

**Comment by saiyam1814** [comment]
LGTM

**Pull request merged**